docs/docs/troubleshooting.md

```diff
@@ -54,7 +54,9 @@ Since then, a much simpler procedure of performing a bluetooth reset for split k
 
 **New Procedure:**
 
-1. Log into Github and download the "settings clear" UF2 image from the [latest build in Github Actions](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain)
+1. Open [latest build in Github Actions](https://github.com/zmkfirmware/zmk/actions?query=workflow%3ABuild+branch%3Amain) and open the first 'result' on that page.
+1. From the next page under "Artifacts", download the `$boardname-settings_reset-zmk` zip file.
+1. Unzip the downloaded file
```

Comment:
```suggestion
1. Unzip the downloaded file.
```

Reply:
I'm good with all of the above changes. Do I need to do anything with these?